### ISSUE TITLE
Column count doesn't match value count 

### ISSUE THREAD

**Issue by ghost** [PrestaShop/PrestaShop#21949]
#### Describe the bug

During upgrade 1769 to 1770 i have this error

Column count doesn't match value count at row 14 and after

```

SQL 1.7.7.0 1136 in 

INSERT IGNORE INTO `ps_hook` (`id_hook`, `name`, `title`, `description`, `position`) 
VALUES (NULL, 'displayAdminOrderTop', 'Admin Order Top', 'This hook displays content at the top of the order view page', '1'), 

(NULL, 'displayAdminOrderSide', 'Admin Order Side Column', 'This hook displays content in the order view page in the side column under the customer view', '1'), 
(NULL, 'displayAdminOrderSideBottom', 'Admin Order Side Column Bottom', 'This hook displays content in the order view page at the bottom of the side column', '1'),
(NULL, 'displayAdminOrderMain', 'Admin Order Main Column', 'This hook displays content in the order view page in the main column under the details view', '1'),
(NULL, 'displayAdminOrderMainBottom', 'Admin Order Main Column Bottom', 'This hook displays content in the order view page at the bottom of the main column', '1'),
(NULL, 'displayAdminOrderTabLink', 'Admin Order Tab Link', 'This hook displays new tab links on the order view page', '1'), (NULL, 'displayAdminOrderTabContent', 'Admin Order Tab Content', 'This hook displays new tab contents on the order view page', '1'),
(NULL, 'actionGetAdminOrderButtons', 'Admin Order Buttons', 'This hook is used to generate the buttons collection on the order view page (see ActionsBarButtonsCollection)', '1'),
(NULL, 'displayFooterCategory', 'Category footer', 'This hook adds new blocks under the products listing in a category/search', '1'),
(NULL, 'displayBackOfficeOrderActions', 'Admin Order Actions', 'This hook displays content in the order view page after action buttons (or aliased to side column in migrated page)', '1'),
(NULL, 'actionAdminAdminPreferencesControllerPostProcessBefore', 'On post-process in Admin Preferences', 'This hook is called on Admin Preferences post-process before processing the form', '1'),
(NULL, 'displayAdditionalCustomerAddressFields', 'Display additional customer address fields', 'This hook allows to display extra field values added in an address form using hook ''additionalCustomerAddressFields''', '1'), 
(NULL, 'displayAdminProductsExtra', 'Admin Product Extra Module Tab', 'This hook displays extra content in the Module tab on the product edit page', '1'),
(NULL, 'actionFrontControllerInitBefore', 'Perform actions before front office controller initialization', 'This hook is launched before the initialization of all front office controllers'),
(NULL, 'actionFrontControllerInitAfter', 'Perform actions after front office controller initialization', 'This hook is launched after the initialization of all front office controllers'),
(NULL, 'actionAdminControllerInitAfter', 'Perform actions after admin controller initialization', 'This hook is launched after the initialization of all admin controllers'), 
(NULL, 'actionAdminControllerInitBefore', 'Perform actions before admin controller initialization', 'This hook is launched before the initialization of all admin controllers'), 
(NULL, 'actionControllerInitAfter', 'Perform actions after controller initialization', 'This hook is launched after the initialization of all controllers'),
(NULL, 'actionControllerInitBefore', 'Perform actions before controller initialization', 'This hook is launched before the initialization of all controllers'),
(NULL, 'actionAdminLoginControllerBefore', 'Perform actions before admin login controller initialization', 'This hook is launched before the initialization of the login controller'), 
(NULL, 'actionAdminLoginControllerLoginBefore', 'Perform actions before admin login controller login action initialization', 'This hook is launched before the initialization of the login action in login controller'), 
(NULL, 'actionAdminLoginControllerLoginAfter', 'Perform actions after admin login controller login action initialization', 'This hook is launched after the initialization of the login action in login controller'),
(NULL, 'actionAdminLoginControllerForgotBefore', 'Perform actions before admin login controller forgot action initialization', 'This hook is launched before the initialization of the forgot action in login controller'),
(NULL, 'actionAdminLoginControllerForgotAfter', 'Perform actions after admin login controller forgot action initialization', 'This hook is launched after the initialization of the forgot action in login controller'),
(NULL, 'actionAdminLoginControllerResetBefore', 'Perform actions before admin login controller reset action initialization', 'This hook is launched before the initialization of the reset action in login controller'), 
(NULL, 'actionAdminLoginControllerResetAfter', 'Perform actions after admin login controller reset action initialization', 'This hook is launched after the initialization of the reset action in login controller')
: Column count doesn't match value count at row 14
```

There is no position 5th argument

#### Steps to Reproduce

Make an upgrade 1769 to 1770 and check SQL error 

#### Additional information

Informations sur votre serveur
Informations sur votre serveur Linux #799710 SMP Wed Jul 10 08:31:28 UTC 2019 x86_64

Version du logiciel serveur : Apache

Version de PHP : 7.2.34
Limite de mémoire : 2048M
Temps maximal d'exécution : 18000
Taille max. pour envoi de fichiers : 10M
info_outline Informations sur la base de données

Version de MySQL : 10.1.45-MariaDB-0+deb9u1
Serveur MySQL : localhost
Nom MySQL : 176
Utilisateur MySQL : 176
Préfixe des tables : ps_
Moteur MySQL : InnoDB
Connecteur MySQL : DbPDO

**Comment by hibatallahAouadni** [comment]
Related to #21948

**Comment by hibatallahAouadni** [comment]
Hello @okom3pom

Could you please modify original issue #21948 as there is no need to open a new one as long as all issues you are having are related. 
And edit the **Fixed ticket?** block on your PR #21953.

Thanks!

**Comment by hibatallahAouadni** [comment]
Duplicate of #21948

**Comment by ghost** [comment]
It's not the same :) or duplicate :) Regards !

**Comment by hibatallahAouadni** [comment]
Ping @matks wdyt?

**Comment by hibatallahAouadni** [comment]
Hello @okom3pom 

As discussed with @PierreRambaud it seems this topic is totally different from this issue #21948
So I reopen it.

PS: sorry for the misunderstanding.

Thanks!

**Comment by ghost** [comment]
@hibatallahAouadni my english is bad, it's difficult to unterdand me; no problem

**Comment by ghost** [comment]
For this one i open a PR

**Comment by Progi1984** [comment]
Bug approved : some lines have position and not another.

So it's a problem for upgrade.

Ping @PrestaShop/product-team

**Comment by hibatallahAouadni** [comment]
Ping @marionf what is the status of this ticket? **`TBS`** or **`To Do`** ?

**Comment by eternoendless** [comment]
Fixed by #21953
